README.md

```diff
@@ -299,22 +296,31 @@ Authentication is based on the [OAuth 2.0 Protocol](http://tools.ietf.org/html/d
 
 **Resource URL (public resource)**
 
-    GET /bcf/auth
+    GET /bcf/{version}/auth
 
 **Parameters**
 
 |Parameter|Type|Description|Required|
 |---------|----|-----------|--------|
-|oauth2_auth_url|string|URL to authorisation page|false|
+|oauth2_auth_url|string|URL to authorisation page (used for Authorization Code Grant and Implicit Grant OAuth2 flows)|false|
 |oauth2_token_url|string|URL for token requests|false|
 |oauth2_dynamic_client_reg_url|string|URL for automated client registration|false|
 |http_basic_supported|boolean|Indicates if Http Basic Authentication is supported|false|
+|supported_oauth2_flows|string[]|array of supported OAuth2 flows|
 
 If `oauth2_auth_url` is present, then `oauth2_token_url` must also be present and vice versa. If properties are not present in the response, clients should assume that the functionality is not supported by the server, e.g. a missing `http_basic_supported` property would indicate that Http basic authentication is not available on the server.
 
+OAuth2 flows are described in detail in the [OAuth2 specification](https://tools.ietf.org/html/rfc6749). BCF API servers may support the following workflows:
+* `authorization_code_grant` - [4.1 - Authorization Code Grant](https://tools.ietf.org/html/rfc6749#section-4.1)
+* `implicit_grant` - [4.2 - Implicit Grant](https://tools.ietf.org/html/rfc6749#section-4.2)
+* `resource_owner_password_credentials_grant` - [4.3 - Resource Owner Password Credentials Grant](https://tools.ietf.org/html/rfc6749#section-4.3)
+* `extension_grants` - [4.5 - Extension Grants](https://tools.ietf.org/html/rfc6749#section-4.5)
```

Comment:
I am not 100% sure how this flow works. Have you verified that it fit our needs?